style.css

```diff
@@ -70,10 +70,10 @@ label {
   font-family: 'Inter', sans-serif;
```

Comment:
- [ ] Kindly make sure your pop-up modal is responsive both on desktop and mobile versions. As you may notice on your mobile design the user is not able to close the pop-up window. Kindly fix that.

Your mobile design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/210383334-d54d8cc1-4435-4642-89eb-a92f132ec598.png)